app/models/school.rb

```diff
@@ -32,4 +32,23 @@ def sierra_code
 
     return match.sierra_code
   end
+
+  
+  # Get all active schools
+  def self.active_schools_data
+    active.map { |school| school.name_id }
```

Comment:
There's a tiny shortcut you can do here. Instead of 
`active.map { |school| school.name_id }`
you can say
`active.map(&:name_id)`